interface/super/edit_layout.php

```diff
@@ -179,28 +265,33 @@ function addOrDeleteColumn($layout_id, $field_id, $add = true)
     for ($lino = 1; isset($fld[$lino]['id']); ++$lino) {
         $iter = $fld[$lino];
         $field_id = formTrim($iter['id']);
-        $data_type = formTrim($iter['data_type']);
+        $field_id_original = formTrim($iter['originalid']);
+        $data_type = formTrim($iter['datatype']);
         $listval = $data_type == 34 ? formTrim($iter['contextName']) : formTrim($iter['list_id']);
+        $action = $iter['action'];
+        if ($action == 'value') $action = 'value=' . $iter['value'];
 
         // Skip conditions for the line are stored as a serialized array.
-        $condarr = array();
-        for ($cix = 0; !empty($iter['condition_id'][$cix]); ++$cix) {
-            $andor = empty($iter['condition_andor'][$cix]) ? '' : $iter['condition_andor'][$cix];
-            $condarr[$cix] = array(
+        $condarr = array('action' => strip_escape_custom($action));
```

Comment:
odd to see strip_escape_custom(). what is this for?

Reply:
In 5.x this is a do-nothing function. I'll omit the call.